config/sidekiq_scheduler.yml

```
@@ -271,3 +271,8 @@ TestUserDashboard::MirrorAccountAvailabilityLogsInBigQuery:
   cron: "0 */2 * * * America/New_York"
   class: TestUserDashboard::MirrorAccountAvailabilityLogsInBigQuery
   description: "Mirrors the test_user_dashboard_tud_account_availability_logs table in BigQuery."
+
+GetNew1095Bs:
+  cron: "0 3 * * * America/New_York"
+  class: Form1095::GetNew1095Bs
+  description: "Getting new 1095Bs"
```

Newline at end of file. 
```suggestion
  description: "Getting new 1095Bs"
  
```